db/migrate/20150227185631_add_users_state_flag.rb

```diff
@@ -0,0 +1,15 @@
+class AddUsersStateFlag < ActiveRecord::Migration
+  def up
+    add_column :users, :state, :string, default: 'temp'
+    execute "update users set state='temp' where is_temp='t'"
```

Comment:
Not quite, "temp" is the default but existing columns won't be set to that, they'll just be null.  But we probably don't want that either, they should be "activated" as you've pointed out.  Will correct shortly.  Thanks Dante